packages/sanity/src/studio/studio-components-hooks/hooks.ts

```diff
@@ -0,0 +1,52 @@
+import {LayoutProps} from 'framer-motion'
```

Comment:
Rename this directory to `componentHooks` 🙏

Reply:
Oh, that's wrong. Importing `LayoutProps` from `framer-motion`

Reply:
Aha, I intended to comment on the directory by selecting the first line. But yeah, that is wrong indeed 😬